src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -131,6 +131,13 @@ private static Option<T> CreateCommonOption<T>(OptionGroup group, string name, T
             defaultValue: false,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferCollectionInitializer_FadeOutCode"));
 
+        internal static readonly PerLanguageOption<CodeStyleOption<bool>> PreferSimplifiedBooleanExpressions = CreateOption(
```

Comment:
Interesting - your edit here looks fine but I just noticed that the field `OperatorPlacementWhenWrapping` below does not use the `CreateOption` helper, which in turn means it will not get added to `AllOptions` builder, and hence not get exposed through `ICodeStyleOptionProvider` and will not serialize into solution snapshot. Can you file a bug or fix it while you are touching this code?

Reply:
done.